images/users/shoesofprey.svg

```diff
@@ -0,0 +1 @@
52 4.17s.75 1.86 1.22 2.38.93.93 1.39 1.16c.35.12.58.23.75.23h-14.55c.17 0 .41-.06.75-.23.46-.17.93-.52 1.39-1.1.46-.52.87-1.33 1.22-2.38.35-1.04.52-2.43.52-4.23v-41.57c0-1.8-.17-3.25-.58-4.29-.41-1.04-.81-1.85-1.27-2.43-.46-.52-.98-.93-1.45-1.1-.41-.17-.7-.29-.87-.35h21.28c3.07 0 5.74.46 8.06 1.39 2.32.93 4.23 2.15 5.74 3.71 1.51 1.57 2.67 3.36 3.48 5.39s1.21 4.12 1.21 6.26zm-7.19.47c0-1.74-.29-3.36-.87-4.99-.58-1.56-1.39-2.96-2.49-4.12s-2.49-2.09-4.17-2.78c-1.68-.69-3.59-1.04-5.74-1.04h-8.41v24.58c.81.29 2.03.52 3.65.7 1.62.17 3.31.23 5.04.23 2.09 0 3.94-.35 5.57-.98 1.62-.64 2.96-1.56 4.06-2.67s1.91-2.44 2.49-4c.58-1.46.87-3.14.87-4.93zM811.6 208.39l-12.12-20.29c-1.16-2.03-2.32-3.36-3.42-4-1.1-.64-2.49-.99-4-.99h-2.15v17.34c0 1.74.17 3.13.52 4.17s.75 1.86 1.16 2.44c.41.58.87.93 1.33 1.1l.75.23h-14.55c.17 0 .41-.06.75-.23.46-.17.93-.52 1.39-1.1.46-.52.87-1.33 1.22-2.38.35-1.04.52-2.43.52-4.23v-41.51c0-1.8-.17-3.19-.58-4.29-.4-1.04-.81-1.91-1.27-2.49s-.98-.93-1.45-1.1-.75-.29-.93-.35h20.29c2.84 0 5.39.41 7.71 1.22 2.32.81 4.29 1.91 5.91 3.36s2.9 3.13 3.83 5.04c.87 1.91 1.33 4 1.33 6.26 0 1.74-.29 3.48-.93 5.28-.58 1.8-1.56 3.42-2.84 4.93-1.27 1.51-2.9 2.78-4.87 3.77-1.97.99-4.35 1.62-7.13 1.86.75.29 1.51.87 2.32 1.68.81.81 1.51 1.8 2.15 2.9l10.03 16.47c1.16 1.85 2.38 3.07 3.65 3.71 1.22.58 2.26.99 3.19 1.22H811.6v-.02zm-.93-41.33c0-1.51-.29-2.96-.81-4.41s-1.39-2.72-2.49-3.83-2.49-2.03-4.12-2.72c-1.62-.7-3.54-1.04-5.68-1.04h-7.88v24h7.88c2.44 0 4.47-.35 6.09-.99s3.02-1.56 4-2.61c1.04-1.1 1.8-2.38 2.26-3.77.52-1.5.75-3.01.75-4.63zM881.23 208.39h-35.94c.17 0 .41-.06.75-.23.46-.17.93-.52 1.45-1.1.46-.58.93-1.39 1.27-2.49.35-1.04.52-2.55.52-4.35v-41.34c0-1.8-.17-3.25-.58-4.29-.4-1.04-.81-1.85-1.33-2.43-.52-.52-1.04-.93-1.56-1.1-.41-.17-.69-.29-.87-.35h34.55v9.33c-.06-.23-.17-.64-.4-1.22-.17-.58-.58-1.16-1.22-1.74-.58-.58-1.51-1.1-2.67-1.51-1.16-.41-2.78-.64-4.81-.64H856.3v21.74h11.77c1.39 0 2.49-.17 3.36-.46.81-.29 1.51-.64 1.97-1.04.46-.4.75-.75.93-1.16.12-.35.23-.64.29-.81v11.36c0-.11-.06-.35-.23-.75-.17-.35-.46-.75-.93-1.22-.46-.41-1.1-.81-1.97-1.1-.81-.29-1.97-.46-3.36-.46h-11.88v23.02h14.55c1.74 0 3.19-.06 4.47-.23 1.22-.17 2.32-.46 3.25-.93s1.68-1.1 2.32-1.85c.64-.75 1.27-1.8 1.91-3.01l-1.52 10.36zM943.49 153.49c0-1.1-.29-1.8-.87-2.14-.52-.29-.93-.46-1.22-.58h13.1c-.17.06-.41.17-.75.35-.46.17-1.04.52-1.74 1.1-.69.58-1.51 1.45-2.49 2.61-.93 1.16-1.97 2.67-3.07 4.58l-14.44 24.76v16.18c0 1.74.17 3.13.52 4.17s.75 1.86 1.16 2.44c.41.52.87.93 1.33 1.16.35.12.58.23.75.23h-14.78c.17 0 .41-.06.75-.23.46-.17.93-.52 1.39-1.1.46-.52.87-1.33 1.22-2.43.35-1.04.52-2.49.52-4.29v-16.12l-14.55-25.04c-1.1-1.85-2.09-3.36-3.01-4.46s-1.74-1.97-2.43-2.55c-.7-.58-1.27-.93-1.74-1.1-.41-.17-.64-.29-.75-.35h15.65c-.12.06-.23.12-.35.18-.17.06-.41.17-.64.35-.23.17-.46.41-.64.75-.17.35-.23.81-.23 1.39 0 .64.17 1.39.58 2.32.35.87.99 2.03 1.8 3.42l11.42 19.65 10.84-18.73c.99-1.68 1.68-3.01 2.15-4 .35-.95.52-1.82.52-2.52zM121.88 151.11v33.28L55.38 251v-99.89h66.5z"/></g></svg>
```

Comment:
Thanks for your PR. Could you resize your SVG to 500x200 as mentionned at the bottom of the user page.